gluon-private-wifi: add support for WPA3 encryption (#1876)
freifunk-gluon/gluon: docs/dev/hardware.rst

although other chipsets may also work. VAP (multiple SSID) support
is a requirement.

.. _device-class-definition:

Device classes
--------------

> Gluon is currently aware of two device classes. Depending on the device class, different

Reviewer: ```suggestion
Gluon currently is aware of two device classes. Depending on the device class, different
```

features can be installed onto the device.

> The ``tiny`` device-class contains the following type of devices:

Reviewer: ```suggestion
The ``tiny`` device-class contains devices with the following limitations:
```
